$MirOS: ports/devel/automake/1.9/patches/patch-lib_am_libs_am,v 1.1.7.1 2005/03/18 15:44:28 tg Exp $
--- lib/am/libs.am.orig	Sat May 14 20:20:44 2005
+++ lib/am/libs.am	Sun Oct  2 11:44:58 2005
@@ -31,7 +31,7 @@ am__installdirs += "$(DESTDIR)$(%NDIR%di
 ?!BASE?%DIR%LIBRARIES_INSTALL = $(install_sh_DATA)
 ?EXEC?.PHONY install-exec-am: install-%DIR%LIBRARIES
 ?!EXEC?.PHONY install-data-am: install-%DIR%LIBRARIES
-install-%DIR%LIBRARIES: $(%DIR%_LIBRARIES)
+install-%DIR%LIBRARIES:
 	@$(NORMAL_INSTALL)
 	test -z "$(%NDIR%dir)" || $(mkdir_p) "$(DESTDIR)$(%NDIR%dir)"
 ## Funny invocation because Makefile variable can be empty, leading to
@@ -58,6 +58,7 @@ install-%DIR%LIBRARIES: $(%DIR%_LIBRARIE
 ?BASE?	    p=$(am__strip_dir) \
 ## Must ranlib after installing because mod time changes.
 	    echo " $(RANLIB) '$(DESTDIR)$(%NDIR%dir)/$$p'"; \
+	    chmod u+w "$(DESTDIR)$(%NDIR%dir)/$$p"; \
 	    $(RANLIB) "$(DESTDIR)$(%NDIR%dir)/$$p"; \
 	  else :; fi; \
 	done
